src/v/features/BUILD

```diff
@@ -21,6 +21,7 @@ redpanda_cc_library(
         "//src/v/cluster:features",
         "//src/v/cluster:version",
         "//src/v/config",
+        "//src/v/metrics",
```

Comment:
I'm not 100% sure if this could be `implementation_deps` only. I'll need to double-check.

Reply:
This point no longer applies. I've made it an `implementation_deps` now that the probe is hidden inside the implementation file.